azext_edge/edge/_help.py

```diff
@@ -1544,13 +1544,11 @@ def load_iotops_help():
         - name: Add a client certificate.
```

Comment:
In the long summary we should explain the dynamic of the command. Explain the auto derivation, and what happens when subject name and app uri are explicitly provided.